apis/coursebatchmanapi/coursebatchmanapi.yaml

```diff
@@ -56,7 +56,843 @@ paths:
+        '500':
+          description: 'INTERNAL SERVER ERROR. We track these errors automatically and try to set it right at the earliest. Try refreshing the page. If the problem persists contact us at info@sunbird.org.'
+          schema:
+            allOf:
+             - $ref: 'https://raw.githubusercontent.com/project-sunbird/project-sunbird.github.io/dev/apis/definitions/envelope.yaml#/ApiResponse'
+             - $ref: '#/definitions/BatchCreateErrorResponse'
+  /batch/user/add/{Batch_Id}:  
+    parameters:
+      - in: path
+        name: Batch_Id
+        type: string
+        required: true
+        description: 'Append a valid Batch ID To the Request URL'   
+    post:
+      description: >-
+        This API is associated with addding a user  to a particular batch on the Sunbird Platform.
+          - The format of the resource URL is '{{host}}/course/v1/batch/user/add/{Batch_Id}'
+          - The **_/batch/user/add/{Batch_Id}_** endpoint executes "Add user to a batch" request to add a registered user to a specific batch.
+          - It is a must to provide values for required parameters 
+      summary: Add User to a specific batch
+      tags:
+      - Course Batch APIs
+      operationId: '{{host}}/course/v1/batch/user/add/{Batch_Id}'
+      produces:
+      - application/json
+      parameters:
+      - name: Body
+        in: body
+        required: true
+        description: >-
+            The body refers to the format of the request.
+              - The body contains metadata about the user to be added to a specific batch
+        schema:
+          allOf:
+             - $ref: 'https://raw.githubusercontent.com/project-sunbird/project-sunbird.github.io/dev/apis/definitions/envelope.yaml#/ApiResponse'
+             - $ref: '#/definitions/AddUserToBatchReq'
+      - name: Content-Type
+        in: header
+        required: true
+        type: string
+        description: >-
+            The Content Type entity is the media type of the resource.Possible media types can be:- 
+              - Application/json
+      - name: ts
+        in: header
+        required: true
+        type: string
+        description: 'The time stamp at which the **Add a user to a specific batch** request was sent. '
+      - name: X-msgid
+        in: header
+        required: true
+        type: string
+        description: 'It is a Unique ID which identifies the request incase the same API is executed multiple times.'
+      - name:  x-authenticated-user-token
+        in: header
+        required: true
+        type: string
+        description: 'The alphanumeric string for accessing the API'
+      - name: Authorization
+        in: header
+        required: true
+        type: string
+        description: To make use of the API, you require authorization. Raise a request to the administrator for the use of the API. You will receive the authorization key. Specify the key received, here.'

```

Comment:
Authorization is mandatory for using an API. Request the administrator for the access key. Enter the key received through email